Flow.Launcher/Helper/WindowsInteropHelper.cs

```diff
@@ -80,38 +55,44 @@ public static bool IsWindowFullscreen()
             return false;
         }
 
-        StringBuilder sb = new StringBuilder(256);
-        GetClassName(hWnd, sb, sb.Capacity);
-        string windowClass = sb.ToString();
+        string windowClass;
+        int capacity = 256;
+        char[] buffer = new char[capacity];
+        fixed (char* pBuffer = buffer)
+        {
+            PInvoke.GetClassName(hWnd, pBuffer, capacity);
+            int validLength = Array.IndexOf(buffer, '\0');
+            if (validLength < 0) validLength = capacity;
+            windowClass = new string(buffer, 0, validLength);
```

Comment:
Could you share some doc related to this? (actually maybe add as comment)

Reply:
64-66 is one solution for trimming the length of  one string. Originally I want to use new string(pBuffer) or buffer.ToString(). But it seems they will have some problems.

Reply:
64-66 is one solution for trimming the length of  one string. Originally I want to use new string(pBuffer) or buffer.ToString(). But it seems they will have some problems.

Reply:
new string(pBuffer):

Why It Might Not Work Here:
While pBuffer is a valid pointer to your character array (buffer), the memory beyond the valid string data might not contain a null terminator if the API doesn't explicitly add one.
If the buffer is not null-terminated, this approach could lead to undefined behavior or garbage data being included in the resulting string.

Reply:
buffer.ToString():

Why It Doesn't Work:
This does not extract or process the contents of the buffer at all. It simply provides a string representation of the type, which is completely unrelated to the buffer's content

Reply:
@taooceros Now add comments for it.